test/module/irohad/consensus/yac/yac_block_storage_test.cpp

```diff
@@ -18,39 +18,39 @@ static logger::Logger log_ = logger::testLog("YacBlockStorage");
 
 class YacBlockStorageTest : public ::testing::Test {
  public:
-  YacHash hash;
-  PeersNumberType number_of_peers;
-  YacBlockStorage storage = YacBlockStorage(
-      YacHash(iroha::consensus::Round{1, 1}, "proposal", "commit"), 4);
+  PeersNumberType number_of_peers = 6;
```

Comment:
`const PeersNumberType kNumberOfPeers = 6;`
the same with the following consts & files